apps/meteor/app/api/server/v1/moderation.ts

```diff
@@ -63,11 +64,65 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'moderation.userReports',
+	{
+		authRequired: true,
+		validateParams: isReportHistoryProps,
+		permissionsRequired: ['view-moderation-console'],
+	},
+	{
+		async get() {
+			try {
+				const { latest: _latest, oldest: _oldest, selector = '' } = this.queryParams;
+
+				const { count = 20, offset = 0 } = await getPaginationItems(this.queryParams);
+
+				const { sort } = await this.parseJsonQuery();
+
+				const latest = _latest ? new Date(_latest) : new Date();
+
+				const oldest = _oldest ? new Date(_oldest) : new Date(0);
+
+				const escapedSelector = escapeRegExp(selector);
+
+				const reports = await ModerationReports.findUserReports(latest, oldest, escapedSelector, {
+					offset,
+					count,
+					sort,
+				}).toArray();
+
+				if (reports.length === 0) {
+					return API.v1.success({
+						reports,
+						count: 0,
+						offset,
+						total: 0,
+					});
+				}
+
+				const total = await ModerationReports.getTotalUniqueReportedUsers(latest, oldest, escapedSelector);
+
+				const result = {
+					reports,
+					count: reports.length,
+					offset,
+					total,
+				};
+				return API.v1.success(result);
+			} catch (error) {
+				SystemLogger.error(error);
+				return API.v1.failure('Error while fetching user reports');
+			}
```

Comment:
this try/catch isn't really needed, since the API code has one already for each endpoint.

Reply:
Thank you, removed!